admin/c2cgeoportal_admin/views/roles.py

```diff
@@ -35,3 +35,7 @@ def view(self):
                  renderer="../templates/edit.jinja2")
     def save(self):
         return super().save()
+
+    @view_config(route_name='c2cgeoform_delete')
```

Comment:
Should be updated accordingly to routes in c2cgeoform after merge of camptocamp/c2cgeoform#117